tests/non-js-resources/subtests/dependencies/subtests/deduping/webpack/webpack.config.js

```diff
@@ -15,12 +15,11 @@ module.exports = {
     rules: [
       {
         test: /\.css$/,
-        enforce: 'pre',
-        use: ['extract-loader', 'css-loader'],
+        use: ['style-loader', 'css-loader'],
```

Comment:
This bundles the CSS into the JS, which is a bit too different from the other implementations.

I tried using the technique from `non-js-resources/subtests/css/subtests/url` to externalize the CSS, but `exports-loader` crashes trying to parse the referenced PNG image using Babel (not sure if that's supposed to work).

Here's what I have working instead (I've used inline require() here because GitHub doesn't allow suggestions on unchanged lines, boo):

```suggestion
        use: [require('mini-css-extract-plugin').loader, 'css-loader'],
```